src/daemon/rpmostreed-os.cxx

```diff
@@ -1415,6 +1449,7 @@ os_handle_cleanup (RPMOSTreeOS *interface,
   if (transaction == NULL)
     goto out;
 
+  rpmostree_transaction_set_title ((RPMOSTreeTransaction*)transaction, "cleanup");
   rpmostreed_sysroot_set_txn (rsysroot, transaction);
```

Comment:
Since these two functions seem to always be paired, perhaps a good future cleanup would be adding
`rpmostreed_sysroot_set_txn_and_title (rsysroot, transaction, "cleanup");` or so?

Reply:
ah, yes. Cleanup here https://github.com/coreos/rpm-ostree/pull/2974